packages/design-system/src/components/Table/TableCaption.test.jsx

```diff
@@ -29,5 +29,7 @@ describe('Table', function () {
     const table = wrapper.find('caption');
     expect(table).toHaveLength(1);
     expect(table.hasClass('ds-c-table__caption')).toBe(true);
+
+    expect(wrapper).toMatchSnapshot();
```

Comment:
Could we add a test for when TableCaption renders the scrollableNotice

Reply:
Not sure how to set viewport size to activate scrollableNotice. Will work on it